src/color.js

```diff
@@ -119,10 +119,6 @@ export default class Color {
 	}
 
 	static defineFunction (name, code, o = code) {
-		if (arguments.length === 1) {
-			[name, code, o] = [arguments[0].name, arguments[0], arguments[0]];
-		}
```

Comment:
Remove the corresponding type definition: https://github.com/LeaVerou/color.js/blob/main/types/src/color.d.ts#L93